app/policies/project_policy.rb

```diff
@@ -6,6 +6,14 @@ def initialize(user, project)
     @project = project
   end
 
+  def index?
+    project.user_id == user.id
+  end
+
+  def create?
+    project.user_id == user.id
```

Comment:
In some cases we can return just `true`. 
For example we have roles for user and user with `owner` role can create record, user with `manager` role can't do this.